solr/solr-ref-guide/modules/deployment-guide/pages/distributed-tracing.adoc

```diff
@@ -51,7 +57,7 @@ https://docs.datadoghq.com/tracing/setup/java/[datadog-java-agent] uses Javaagen
 The `modules/jagertracer-configurator` provides a default implementation for setting up Jaeger Tracer.
 
 Add the solr-jaegertracer JAR file and the other JARs provided with this module to your Solr installation, ideally to each node.
-GSON is a dependency that is only used by Jaeger's "remote" sampler,
+https://github.com/google/gson[GSON] is a dependency that is only used by Jaeger's "remote" sampler,
```

Comment:
Good darn question...  At first I assumed it was a licensing issue, but it appears to be ASL2...   Probably worth a JIRA!

Reply:
If you do `gradlew dev` on main you indeed get gson included in `modules/jaegertracer-configrator/lib`.
![Skjermbilde 2022-06-02 kl  13 46 13](https://user-images.githubusercontent.com/409128/171622523-5ea38e4b-5d4c-41a5-a53b-24e269560953.png)

Reply:
interesting...  I've never used the jaeger stuff, but may give it a whirl on adding to https://github.com/querqy/chorus

Reply:
So the exclude stuff in https://github.com/apache/solr/blob/main/solr/modules/jaegertracer-configurator/build.gradle#L29 does not work then. Does it use wrong coordinates?

Reply:
@dsmiley you touched this code some time ago. Why would we not like gson shipped with Solr? Can we simply remove the exlusion from build.gradle, and remove the note from ref-guide?

Reply:
Because of the note I left:
> GSON is a dependency that is only used by Jaeger's "remote" sampler

We should not include dependencies for optional things we don't expect someone is likely to use.  The user can add it if they rally want to.

Reply:
So, reading through https://www.jaegertracing.io/docs/1.34/sampling/ that is why you would need GSON....    We could clarify the docs a bit (or reference https://www.jaegertracing.io/docs/1.34/sampling/) and then fix the build.gradle to not include gson.

Reply:
I added [SOLR-16236](https://issues.apache.org/jira/browse/SOLR-16236) so cover this, and going to resolve this conversation ;-)